website/docs/d/federated_settings_identity_provider.html.markdown

```diff
@@ -46,6 +46,8 @@ In addition to all arguments above, the following attributes are exported:
 
 ### FederatedSettingsIdentityProvider
 
+* `description` - The description of the identity provider.
+* `authorization_type` - Indicates whether authorization is granted based on group membership or user ID. Either `GROUP` or `USER`.
```

Comment:
for the plural resource we use slightly different verbiage: Valid values are GROUP or USER. I think we should standardize on that for consistency.

Reply:
Changed it to be consistent. Thanks for the input